src/transformer/encode.ts

```diff
@@ -552,33 +543,21 @@ const encoders: NodeEncoderMap = {
         : false,
   }),
   equation: (node): Partial<Equation> => ({
-    MathMLStringRepresentation:
-      node.attrs.MathMLStringRepresentation || undefined,
-    TeXRepresentation: node.attrs.TeXRepresentation,
-    SVGStringRepresentation: node.attrs.SVGStringRepresentation,
-    // title: 'Equation',
+    content: node.attrs.content,
```

Comment:
we need a json-schema update to remove `MathMLStringRepresentation`, `TeXRepresentation`, `SVGStringRepresentation` and replace them with content

Reply:
I already have it locally, but there still other things I'm waiting a confirmation which might also affect the json-schema